js_basics/fibonacci_sequence/index.js

```diff
@@ -0,0 +1,16 @@
+const seqNum = process.argv.slice(2);
+
+if (!seqNum[0]) {
+    console.log("A number arguement is required");
+    process.exit(1);
+} else if (seqNum[0] < 0) {
+    console.log("An integer number is required.")
+    process.exit(1);
+}
+let fibo = [];
+fibo[0] = 1;
+fibo[1] = 1;
+for (let i = 2; i < seqNum[0]; i++) {
+    fibo[i] = fibo[i - 1] + fibo[i - 2];
+}
+console.log(fibo.slice(0, seqNum[0]));
```

Comment:
This solution is logging out an array with the numbers. We want only the numbers to be outputted

Reply:
Also make sure there's an empty line at the end of files